azure-iot-hub-devicesdk/samples/receive_c2d_message.py

```diff
@@ -41,7 +41,7 @@ def c2d_message_handler(c2d_message):
 
 while True:
     selection = input("Press Q: Quit for exiting\n")
-    if selection is "Q" or selection is "q":
+    if selection == "Q" or selection == "q":
```

Comment:
VSTS Flake8 plugin seems to have updated as of Jan 30th such that this change had to be made in order to pass static code analysis.